pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -62,8 +64,15 @@ buildPythonPackage rec {
     let
       tcl_tk_cache = ''"${tk}/lib", "${tcl}/lib", "${lib.strings.substring 0 3 tk.version}"'';
     in
-    lib.optionalString enableTk
-      "sed -i '/self.tcl_tk_cache = None/s|None|${tcl_tk_cache}|' setupext.py";
+    lib.optionalString enableTk ''
+      sed -i '/self.tcl_tk_cache = None/s|None|${tcl_tk_cache}|' setupext.py
+    '' +
+    lib.optionalString stdenv.isLinux ''
```

Comment:
Shoud this be `stdenv.isLinux && (enableGtk3 || enableTk || enableQt)`?

Reply:
Right.